keras_cv/models/mobilenet_v2.py

```diff
@@ -0,0 +1,382 @@
+
+    For transfer learning use cases, make sure to read the [guide to transfer
+        learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        alpha: controls the width of the network. This is known as the
+            depth multiplier in the MobileNetV3 paper, but the name is kept for
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        include_rescaling: whether or not to Rescale the inputs.If set to True,
+            inputs will be passed through a `Rescaling(scale=1.0 / 127.5, offset=-1.0)`
+            layer, defaults to True.
+        include_top: whether to include the fully-connected layer at the top of the
+            network.  If provided, `classes` must be provided.
+        weights: one of `None` (random initialization), or a pretrained weight file
+            path.
+        pooling: optional pooling mode for feature extraction
+            when `include_top` is `False`.
+            - `None` means that the output of the model will be the 4D tensor output
+                of the last convolutional block.
+            - `avg` means that global average pooling will be applied to the output
+                of the last convolutional block, and thus the output of the model will
+                be a 2D tensor.
+            - `max` means that global max pooling will be applied.
+        num_classes: optional number of classes to classify images into, only to be
+            specified if `include_top` is True, and if no `weights` argument is
+            specified.
+        classifier_activation: the activation function to use, defaults to softmax.
+        name: (Optional) name to pass to the model. Defaults to "MobileNetV3".
+
+    Returns:
+        A `keras.Model` instance.
+
+    Raises:
+        ValueError: if `weights` represents an invalid path to weights file and is not
+            None.
+        ValueError: if `include_top` is True and `num_classes` is not specified.
+    """
+
+    channel_axis = -1
+
+    if weights and not tf.io.gfile.exists(weights):
+        raise ValueError(
+            "The `weights` argument should be either "
+            "`None` or the path to the weights file to be loaded. "
+            f"Weights file not found at location: {weights}"
+        )
+
+    if include_top and not num_classes:
+        raise ValueError(
+            "If `include_top` is True, "
+            "you should specify `num_classes`. "
+            f"Received: num_classes={num_classes}"
+        )
+
+    input = layers.Input(shape=input_shape)
+    x = None
+
+    if include_rescaling:
+        x = layers.Rescaling(scale=1.0 / 127.5, offset=-1.0)(input)
```

Comment:
We should normalize by 1/255

Reply:
The original `keras.applications` uses this rescaling in the preprocessing step, please see:

https://github.com/keras-team/keras/blob/v2.9.0/keras/applications/mobilenet_v2.py#L517

https://github.com/keras-team/keras/blob/07e13740fd181fc3ddec7d9a594d8a08666645f6/keras/applications/imagenet_utils.py#L191-L194

Thus, I thought to go ahead with this

Reply:
Considering https://github.com/keras-team/keras-cv/pull/526#discussion_r923807778 , I will use 1/255 for this